src/code_8001F980.c

```diff
@@ -714,32 +719,33 @@ void convert_to_fixed_point_matrix(Mtx *fixedPointMatrix, Mat4 arg1) {
     fixedPointMatrix->m[3][3] = ((s32) (arg1[3][2] * someMultiplier) << 0x10)      | ((s32) (arg1[3][3] * someMultiplier) & 0xFFFF);
 }
 
-s32 func_800224F0(s16 *arg0, s16 arg1, s16 arg2) {
+bool move_to_reach_angle(s16 *angle, s16 targetAngle, s16 step) {
```

Comment:
`translate_to_angle` ?

Reply:
it's a function who start from an angle and substract a step until it reach a target angle

Reply:
I think this is more like

has_reached_angle

Reply:
I don't has_reach because this doesn't mean if the angle you give in argument have change or not

Reply:
`adjust_angle` ?